pkg/controller/kubemanager/kubemanager_controller.go

```diff
@@ -220,6 +220,11 @@ func (r *ReconcileKubemanager) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	_, err = instance.CreateConfigMap(request.Name+"-"+instanceType+"-configmap-envs", r.Client, r.Scheme, request)
```

Comment:
Nit: I would create a variable, for example `envVariablesConfigMap := request.Name+"-"+instanceType+"-configmap-envs"` and use it here and in line 436